components/dashboard/src/user-settings/Integrations.tsx

```diff
@@ -364,13 +367,15 @@ function GitIntegrations() {
         | undefined
     >(undefined);
 
-    useEffect(() => {
-        updateOwnAuthProviders();
-    }, []);
-
-    const updateOwnAuthProviders = async () => {
-        setProviders(await getGitpodService().server.getOwnAuthProviders());
-    };
+    const {
```

Comment:
swapped this to an ad-hoc `useQuery` to have some loading state, and avoid reloading a bit.